### DISCUSSION TITLE
gh-pages down, but github status says there are no problems

### DISCUSSION THREAD

**Discussion by bradbell** [community/community#22555] [Other Feature Feedback, Questions, & Ideas]
<p>If I check the current github status on the web page<br>
<a href="https://www.githubstatus.com/" rel="nofollow noopener">https://www.githubstatus.com/</a><br>
I get All Systems Operational</p>
<p>I can load a copy of the repository using:<br>
git clone <a href="https://github.com/coin-or/CppAD.git" rel="nofollow noopener">https://github.com/coin-or/CppAD.git</a> cppad.git<br>
cd cppad.git</p>
<h2>I can see the log message for the last two commits<br>
on the gh-pages branch using:<br>
git checkout gh-pages<br>
git log -2<br>
This results in the following output:</h2>
<p>cppad.git&gt;git log -2<br>
commit 002bec33efc3ecd75f4da49623217ae95dde3a35 (HEAD -&gt; gh-pages, origin/gh-pages)<br>
Date:   Sat Jan 5 08:47:51 2019 -0700<br>
    update gh-pages to version 20190105</p>
<p>commit fbdab110ab03bdd453b406d8ce03d624c5cabeb0<br>
Date:   Sun Dec 30 16:06:55 2018 -0700<br>
    update gh-pages to version 20181230</p>
<p>It is currently over 8 hourse since the Jan 5th commit.</p>
<p>I can retrieive a copy of web display of the file doc/cppad.htm using the command:<br>
wget <a href="http://coin-or.github.io/CppAD/doc/cppad.htm" rel="nofollow noopener">http://coin-or.github.io/CppAD/doc/cppad.htm</a></p>
<h2>The command above gets the remote file doc/cppad.htm and writes it as<br>
cppad.htm in the top directory for the git repository.<br>
If I then check for a difference with the current doc/cppad.htm file I get:<br>
diff cppad.htm doc/cppad.htm<br>
cppad.git&gt;diff cppad.htm doc/cppad.htm<br>
15c15<br>
&lt; &lt;title&gt;cppad-20181230: A C++ Algorithmic Differentiation Package&lt;/title&gt;</h2>
<p>&gt; &lt;title&gt;cppad-20190105: A C++ Algorithmic Differentiation Package&lt;/title&gt;<br>
17,18c17,18</p>
<p>… snip …</p>
<p>If I then checkout the previous version, I get no difference with the following commands:<br>
git checkout gh-pages~1<br>
git diff cppad.htm doc/cppad.htm</p>

**Comment by bradbell** [comment, marked as answer]
<p>I checked in a new version of gh-pages and it updated the web page display at</p>
<p><a href="http://coin-or.github.io/CppAD/doc/cppad.htm" rel="nofollow noopener">http://coin-or.github.io/CppAD/doc/cppad.htm</a></p>
<p>right away.</p>
<p>I think that something was wrong at the time of the previous commit to my gh-pages branch and so github did not know to update the web page display.</p>

**Comment by xmnlab** [comment]
sorry for commenting in a very old discussion, but I have the same issue.

all my github pages are down, right now.

the output is:

```
This site can’t be reachedCheck if there is a typo in opensciencelabs.org.
DNS_PROBE_FINISHED_NXDOMAIN
```

I saw that there was some issues with github pages a few days ago .. not sure if I need to apply any change in order to have it up and running again.

**Reply by xmnlab** [reply]
sorry .. it seems that this issues is happening in the dns provider
